app/validators/claim/base_claim_validator.rb

```diff
@@ -91,6 +92,8 @@ def validate_transfer_court(force: false)
 
   def validate_transfer_case_number
     return if @record.errors[:transfer_case_number].present?
+    validate_pattern(:transfer_case_number, URN_PATTERN, 'invalid')
+    return unless looks_like_a_case_number?(:transfer_case_number)
```

Comment:
why not use [guard clause as you have here](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/3231/files#diff-90d4e3266775fe3632d22743de6bc3f4R78)

Reply:
I couldn't because I kept getting into a circle of rubocop linelength/guard clause errors. Adding the return unless looks_like_a_case_number? was the suggested solution here https://github.com/rubocop-hq/rubocop/issues/7215.

Reply:
its more about consistency as you are using guard clause at line `78` but not here

Reply:
As discussed, have amended line 78 so that the validation for case_number and transfer_case_number is consistent.